fw/rbcx-coprocessor/platformio.ini

```diff
@@ -36,3 +35,35 @@ extra_scripts =
 
 debug_tool = stlink
 upload_protocol = stlink
+
+[env:hw10_debug]
+build_type = debug
+build_flags =
+    ${env.build_flags}
+    -DRBCX_HW_VER=0x0100
+    -DUSBD_DP_PORT=GPIOA
+    -DUSBD_DP_PIN=8
+
+[env:hw10_release]
+build_type = release
+build_flags =
+    ${env.build_flags}
+    -DRBCX_HW_VER=0x0100
+    -DUSBD_DP_PORT=GPIOA
+    -DUSBD_DP_PIN=8
+
+[env:hw11_debug]
+build_type = debug
+build_flags =
+    ${env.build_flags}
+    -DRBCX_HW_VER=0x0101
+    -DUSBD_DP_PORT=GPIOD
+    -DUSBD_DP_PIN=2
+
+[env:hw11_release]
+build_type = release
+build_flags =
+    ${env.build_flags}
+    -DRBCX_HW_VER=0x0101
+    -DUSBD_DP_PORT=GPIOD
+    -DUSBD_DP_PIN=2
```

Comment:
Nejradši bych neměl debug/release, ale jen debug. Perf zisk není velkej, ale složitosti celku to přidá.